modules/acm.tf

```diff
@@ -0,0 +1,22 @@
+resource "aws_acm_certificate" "this" {
+  count = local.has_domain ? 1 : 0
+
+  provider = aws.us-east-1
+
+  domain_name               = var.domain
+  validation_method         = "DNS"
+  subject_alternative_names = local.subdomains
+
+  lifecycle {
+    create_before_destroy = true
+  }
+}
+
+resource "aws_acm_certificate_validation" "this" {
+  count = local.has_domain ? 1 : 0
+
+  provider = aws.us-east-1
```

Comment:
Same question here about region

Reply:
If we want to create certs to be associated with CloudFront they must exist in us-east-1. Its just the certs that exist in that region, Historically we have been doing that for our older CloudFront endpoints, which I see was done manually before.